### ISSUE TITLE
[SUGGESTION] Add attacker expression into projectile events

### ISSUE THREAD

**Issue by Fusezion** [SkriptLang/Skript#5783]
### Suggestion

Add the `attacker` expression into events like `projectile hit` and `projectile collide`

### Why?

Ran into someone on discord who was using `attacker` in the events and while this is maybe incorrect way of seeing this I believe `shooter` should be treated as `attacker` in these events

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this suggestion.

**Comment by TheLimeGlass** [comment]
I don't agree with this suggestion, an attacker refers to something damaging another thing. A projectile hit event does not reflect damage, it reflects the projectile colliding with a block or entity. That doesn't always infer damage.
